cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -187,7 +198,7 @@ users:
 	// plugin config
```

Comment:
keep a test in place exercising plugin config with v1beta1 config

Reply:
Done.